nestkernel/connector_model_impl.h

```diff
@@ -180,6 +180,14 @@ GenericConnectorModel< ConnectionT >::used_default_delay()
         kernel().connection_builder_manager.get_delay_checker().assert_valid_delay_ms(
           default_connection_.get_delay() );
       }
+      // Let connections without delay contribute to the delay extrema with wfr_comm_interval
+      // For those connections the min_delay is important as it determines the length
+      // of the global communication interval.
+      else
+      {
+        kernel().connection_builder_manager.get_delay_checker().assert_valid_delay_ms(
+          kernel().simulation_manager.get_wfr_comm_interval() );
```

Comment:
If I understand correctly, the `wfr_comm_interval` is set only during `SimulationManager::set_status` and cannot change otherwise. If yes, then we need to check for a valid value only once (not for every 'delay-less' connection) when it is changed in the SimulationManager or when when `min/max_delay` is change in the `ConnectionBuilderManager`. Also relevant for the subsequent changes in this file.